StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/FeedbackGeneratorAdapter.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  FeedbackGeneratorAdapter.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 3/6/24.
+//
+
+import Foundation
+import UIKit
+
+final class FeedbackGeneratorAdapter {
```

Comment:
there's some downsides to nit pick about this PR: ex. I am not using `prepare()` and I am sometimes scattering this class to be used deep into reusable classes (rather than using DI or else), but I don't see any _practical_ / concrete issues and this minimizes the code impact (very easy to implement), so so far so good 👍